ACCESSIBILITY.md

```diff
@@ -0,0 +1,236 @@
+# Form Accessibility
+
+Forms can present a challenge when it comes to accessibility, most specifically in terms of Assitive Technology (in this case screen readers, which we will refer to as 'AT' going forward).
+
+`o-forms` has been tested with our [contrast checker](https://registry.origami.ft.com/components/o-colors#demo-contrast-ratio-checker) to make sure that our colour contrasts meet [WCAG](https://www.w3.org/WAI/standards-guidelines/wcag/) requirements, so this document is going to walk through `aria` attributes we recommend using.
+
+### Table of Contents:
+- [Single Inputs](#single-inputs)
+- [Multiple Inputs](#multiple-inputs)
+- [Form Validation](#form-validation)
+
+## Single inputs
+`o-forms` considers these types of input 'single input fields`:
+- `input[type=text]` 
+- `input[type=password]`
+- `select`
+- `textarea`
+
+The base structure for all of these types of input is as follows:
+```html
+<label>
+	<span class="o-forms-title">
+		<span class="o-forms-title__main">
+			Label to describe the input here
+		</span>
+	</span>
+
+	<span class="o-forms-input ...modifiers">
```

Comment:
```suggestion
	<span class="o-forms-input">
```

How would you feel about removing this? Would like to avoid pretend markup confusing anyone.

Reply:
I don't really know... I think the information should be there, but maybe as a comment instead